ToString panics

**Describe the bug**
`eris.ToString` panics. Latest master.
BTW, `eris.ToString` isn't available in latest released version of library, that's why I've switched to master.

**To Reproduce**
```golang
func main() {
	err := eris.New("oops")
	err = eris.Wrap(err, "oops 2")

	fmt.Println(eris.ToString(err, true))
}
```
**Expected behavior**
Stacktrace.

**Actual behavior**
```
$ go run main.go
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/rotisserie/eris.(*Stack).insertFrame(0xc00007ec28, 0xc0000a40f0, 0x1, 0x1)
	/Users/dobegor/go/pkg/mod/github.com/rotisserie/eris@v0.1.2-0.20200117012742-876c5063bd3b/stack.go:25 +0x805
github.com/rotisserie/eris.(*UnpackedError).unpackWrapErr(0xc00007ec18, 0xc0000a40c0)
	/Users/dobegor/go/pkg/mod/github.com/rotisserie/eris@v0.1.2-0.20200117012742-876c5063bd3b/format.go:234 +0x2b4
github.com/rotisserie/eris.Unpack(0x111e300, 0xc0000a40c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/dobegor/go/pkg/mod/github.com/rotisserie/eris@v0.1.2-0.20200117012742-876c5063bd3b/format.go:51 +0x168
github.com/rotisserie/eris.ToCustomString(0x111e300, 0xc0000a40c0, 0x1, 0x10ff8b7, 0x1, 0x10ff8b8, 0x1, 0x10ff8ab, 0x1, 0x10ff8b7, ...)
	/Users/dobegor/go/pkg/mod/github.com/rotisserie/eris@v0.1.2-0.20200117012742-876c5063bd3b/format.go:73 +0x50
github.com/rotisserie/eris.ToString(0x111e300, 0xc0000a40c0, 0x10ffd01, 0x6, 0x111e300)
	/Users/dobegor/go/pkg/mod/github.com/rotisserie/eris@v0.1.2-0.20200117012742-876c5063bd3b/format.go:113 +0x1b3
main.main()
	/Users/dobegor/coding/email/main.go:163 +0xbe
exit status 2
```

--- a/format.go
+++ b/format.go
@@ -87,6 +87,7 @@ func ToCustomString(err error, format Format) string {
 			str += format.ErrorSep
 		}
 		str += eLink.formatStr(format)
+		str += format.MsgStackSep
 	}
 
 	if upErr.ExternalErr != "" {

--- a/stack.go
+++ b/stack.go
@@ -16,6 +16,7 @@ func (s *Stack) insertFrame(wFrames []StackFrame) {
 	} else if len(wFrames) == 1 {
 		// append the frame to the end if there's only one
 		*s = append(*s, wFrames[0])
+		return
 	}
 
 	for at, f := range *s {

